foxy/distribution.yaml

```diff
@@ -3505,6 +3505,21 @@ repositories:
         release: release/foxy/{package}/{version}
       url: https://github.com/PickNikRobotics/picknik_ament_copyright-release.git
       version: 0.0.1-2
+  pinocchio:
+    doc:
+      type: git
+      url: https://github.com/stack-of-tasks/pinocchio.git
```

Comment:
The license file is `BSD-2`, but the `package.xml` says `BSD`, which has widely been used as `BSD-3` in the ros community.
It would be nice to update the package.xml license tag to say `BSD-2`, so which license is being used is clear.

Reply:
Thanks for spotting this! I've amended it in the source repository (https://github.com/stack-of-tasks/pinocchio/commit/edd40aca6d4356ad6e2f8663729281318606b633) and we'll include it in the next tagged release (to all distributions)